spec/support/features/interaction_helper.rb

```diff
@@ -11,8 +11,8 @@ def wait_for_content_to_disappear
   rescue Selenium::WebDriver::Error::StaleElementReferenceError
     # A StaleElementReferenceError means that the context the element
     # was in has disappeared, which means the element is gone too.
-  rescue Selenium::WebDriver::Error::UnknownError => e
-    # We sometimes see "UnknownError" with an error message that is similar to a
+  rescue Selenium::WebDriver::Error::GeneralError => e
+    # We sometimes see "GeneralError" with an error message that is similar to a
```

Comment:
I don't think this is the same sort of "UnknownError" we're trying to update in the frontend code, and I'm not sure I'd expect to see it change.